src/Atata/Components/Fields/EditableField`2.cs

```diff
@@ -17,7 +17,7 @@ protected EditableField()
 
         /// <summary>
         /// Gets the <see cref="DataProvider{TData, TOwner}"/> instance for the value indicating whether the control is read-only.
-        /// By default checks "readonly" attribute of scope element.
+        /// By default checks <c>"readonly"</c> attribute of scope element.

```

Comment:
In this case we can skip double quotes for readonly.

```suggestion
        /// By default checks <c>readonly</c> attribute of scope element.
```